entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntityServiceTests.kt

```diff
@@ -322,7 +325,7 @@ class EntityServiceTests {
         val payload =
             """
             {
-              "location": {
+              "observationSpace": {
```

Comment:
?

Reply:
I'm not going to change the test but create one